### ISSUE TITLE
Encountering segfaults when using InterpolatedImage with drawImage

### ISSUE THREAD

**Issue by jchiang87** [GalSim-developers/GalSim#1079]
I'm seeing this when I use the default `x_intepolant` of `quintic`, but `linear` and `cubic` seem ok.
Here's a script to reproduce:
```
import sys
import numpy
import galsim

for interpolant in 'linear cubic quintic'.split():
    print(interpolant)
    sys.stdout.flush()
    fits_image = galsim.InterpolatedImage('GLAGN_host_427_0_disk.fits',
                                          scale=0.04, x_interpolant=interpolant)
    fits_image.withFlux(529.2666077544975)

    gsobj = galsim.Convolve(fits_image, galsim.Kolmogorov(lam_over_r0=0.7763974062349864), galsim.Gaussian(sigma=0.18368260871093112)) * 4.7367431900462575

    image = galsim.Image(bounds=galsim.BoundsI(1391,1440,3416,3465), dtype=numpy.float32)

    gsobj.drawImage(method='phot', image=image, add_to_image=True)
```
and here's the FITS file (gzipped so that github can recognize the file type) that shows the problem:
[GLAGN_host_427_0_disk.fits.gz](https://github.com/GalSim-developers/GalSim/files/4612631/GLAGN_host_427_0_disk.fits.gz)
This is for galsim 2.2.1.

**Comment by rmjarvis** [comment]
On my machine it seg faults for cubic too.  I'm on the case.

**Comment by rmjarvis** [comment]
Fixed in #1080

**Comment by rmjarvis** [comment]
v2.2.4 released with this fix.
